pkgs/applications/networking/cluster/kubevpn/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "kubevpn";
+  version = "1.1.34";
+
+  src = fetchFromGitHub {
+    owner  = "KubeNetworks";
+    repo   = "kubevpn";
+    rev    = "v${version}";
+    sha256 = "sha256-P4lROZ6UxsCtMwGWIDBkXjd8v/wtD7u9LBoUUzP9Tz0=";
+  };
+
+  vendorHash = "sha256-LihRVqVMrN45T9NLOQw/EsrEMTSLYYhWzVm+lYXtFRQ=";
+
+  # TODO investigate why some config tests are failing
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Create a VPN and connect to Kubernetes cluster network, access resources, and more";
+    homepage = "https://github.com/KubeNetworks/kubevpn";
+    license = licenses.mit;
+    maintainers = with maintainers; [ mig4ng ];
```

Comment:
Maybe you also want to add a changelog entry like so:
```nix
changelog = "https://github.com/KubeNetworks/kubevpn/releases/tag/${srv.rev}";
```

Reply:
Added with `src.rev`, thanks for the suggestion 👍